examples/sparse/gcnii.py

```diff
@@ -0,0 +1,179 @@
+        lamda,
+        alpha,
+        dropout=0.5,
+    ):
+        super().__init__()
+        self.hidden_size = hidden_size
+        self.n_layers = n_layers
+        self.lamda = lamda
+        self.alpha = alpha
+
+        self.FC_layers = nn.ModuleList()
+        self.FC_layers.append(nn.Linear(in_size, hidden_size))
+        self.FC_layers.append(nn.Linear(hidden_size, out_size))
+
+        self.CONV_layers = nn.ModuleList()
+        for _ in range(n_layers):
+            self.CONV_layers.append(GCNIIConvolution(hidden_size, hidden_size))
+        self.activation = nn.ReLU()
+        self.dropout = dropout
+
+    def forward(self, A_norm, feature):
+        H = feature
+        H = F.dropout(H, self.dropout)
+        H = self.FC_layers[0](H)
+        H = self.activation(H)
+        H0 = H
+        for i, conv in enumerate(self.CONV_layers):
+            H = F.dropout(H, self.dropout)
+            H = conv(A_norm, H, H0, self.lamda, self.alpha, i + 1)
+            H = self.activation(H)
+        H = F.dropout(H, self.dropout)
+        H = self.FC_layers[-1](H)
+
+        return F.log_softmax(H, dim=1)
+
+
+def evaluate(g, pred):
+    label = g.ndata["label"]
+    val_mask = g.ndata["val_mask"]
+    test_mask = g.ndata["test_mask"]
+
+    # Compute accuracy on validation/test set.
+    val_acc = (pred[val_mask] == label[val_mask]).float().mean()
+    test_acc = (pred[test_mask] == label[test_mask]).float().mean()
+    return val_acc, test_acc
+
+
+def train(args, model, g, A_norm, H):
+    label = g.ndata["label"]
+    train_mask = g.ndata["train_mask"]
+    optimizer = Adam(model.parameters(), args.lr, weight_decay=args.wd)
+
+    loss_fcn = nn.NLLLoss()
+
+    for epoch in range(args.epochs):
+        model.train()
+        optimizer.zero_grad()
+
+        # Forward.
+        logits = model(A_norm, H)
+
+        # Compute loss with nodes in the training set.
+        loss = loss_fcn(logits[train_mask], label[train_mask])
+
+        # Backward.
+        loss.backward()
+        optimizer.step()
+
+        # Compute prediction.
+        pred = logits.argmax(dim=1)
+
+        # Evaluate the prediction.
+        val_acc, test_acc = evaluate(g, pred)
+        if epoch % 20 == 0:
+            print(
+                f"In epoch {epoch}, loss: {loss:.3f}, val acc: {val_acc:.3f}"
+                f", test acc: {test_acc:.3f}"
+            )
+
+
+if __name__ == "__main__":
+    # Training settings
+    argparse = argparse.ArgumentParser()
+    argparse.add_argument("--epochs", type=int, default=1500)
+    argparse.add_argument("--lr", type=float, default=0.01)
+    argparse.add_argument("--wd", type=float, default=5e-4)
+    argparse.add_argument("--n_layers", type=int, default=64)
+    argparse.add_argument("--hidden_size", type=int, default=64)
+    argparse.add_argument("--dropout", type=float, default=0.5)
+    argparse.add_argument("--alpha", type=float, default=0.2)
+    argparse.add_argument("--lamda", type=float, default=0.5)
```

Comment:
Simplify the implementations to set default values whenever needed and remove `argparse`.

Reply:
remove the argparse, add the config init in main function